examples/dm/topology.example.yaml

```diff
@@ -34,7 +35,9 @@ master_servers:
       # rpc-rate-limit: 10.0
       # rpc-rate-burst: 40
   - host: 10.0.1.18
+    name: master2
```

Comment:
Can we set a default name for master node?

Reply:
will set as some kind like {host}:{port} as default if it's empty, but I recommend to set it explicitly by the user.